packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -164,9 +164,12 @@ export const BlockedMessageContent = (props: BlockedMessageContentProps): JSX.El
 };
 
 const extractContentForAllyMessage = (props: ChatMessageContentProps): string => {
-  if (props.message.content) {
+  let attachments = undefined;
+  /* @conditional-compile-remove(attachment-download) */
+  attachments = props.message.attachments;
+  if (props.message.content || attachments) {
```

Comment:
don't need a separate variable or conditional compile here
`if (props.message.content || props.message.attachments) {`

Reply:
props.message doesn't have attachments in stable, so it won't work

Reply:
![image](https://github.com/Azure/communication-ui-library/assets/109105353/92764221-828c-42c0-ab85-a51e2e23ee6c)